apps/mm2txt/README.md

```
@@ -1,4 +1,137 @@
+
+Export only 2D coordinates and intensity:
+
+```bash
+mm2txt mymap.mm --export-fields "x,y,intensity"
+```
+
+## Output
+
+The tool creates separate TXT files for each exported point cloud layer. Files are named using the pattern:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)